tools/Utilities/StatusCheckLogger.cs

```diff
@@ -10,7 +10,7 @@ namespace Utilities;
 /// <param name="Id">The error message ID</param>
 /// <param name="StartLine">The start line (index from 1)</param>
 /// <param name="EndLine">The end line (index from 1)</param>
-public record Diagnostic(string file, int StartLine, int EndLine, string Message, string Id);
+public record StatusCheckMessage(string file, int StartLine, int EndLine, string Message, string Id);
```

Comment:
I changed the name of this record type because the tester uses the "Diagnostic" class from the roslyn APIs. We don't use this in too many places, so the global change seemed the right way to go